packages/nimble-components/src/text-field/styles.ts

```diff
@@ -118,7 +117,11 @@ export const styles = css`
 
     slot[name='start']::slotted(*) {
         flex: none;
-        ${iconColor.cssCustomProperty}: currentcolor;
+        opacity: 0.6;
```

Comment:
Just checking, but this means that _any_ content put in the start slot (including plain text) would get the 60% opacity applied to it. I'm guessing this is fine, as our guidance is that only icons should go in this slot. If we don't have explicit guidance along these lines should we add that to the docs?

Reply:
Yes, that is correct. Any content put in the start slot would get these opacities applied.

Our docs say _To place an icon at the far-left of the text-field, set `slot="start"` on the icon._ Do you think that's explicit enough?

Reply:
Yeah, it's probably fine. Ultimately, I expect our docs to evolve to describe usage guidance at a higher level than in those Storybook settings, which aren't as discoverable.